concourse/pipelines/create-deployer.yml

```diff
@@ -0,0 +1,299 @@
+            ls -l vpc-terraform-outputs/vpc-terraform-outputs.yml
+            cat vpc-terraform-outputs/vpc-terraform-outputs.yml
+
+    - task: terraform-apply
+      config:
+        image: docker:///governmentpaas/docker-terraform
+        inputs:
+        - name: paas-cf
+        - name: vpc-terraform-outputs
+        params:
+          DEPLOY_ENV: {{deploy_env}}
+          AWS_DEFAULT_REGION: {{aws_region}}
+          AWS_ACCESS_KEY_ID: {{aws_access_key_id}}
+          AWS_SECRET_ACCESS_KEY: {{aws_secret_access_key}}
+          TF_VAR_AWS_ACCESS_KEY_ID: {{aws_access_key_id}}
+          TF_VAR_AWS_SECRET_ACCESS_KEY: {{aws_secret_access_key}}
+          TF_VAR_concourse_db_password: {{concourse_db_password}}
+        run:
+          path: sh
+          args:
+          - -e
+          - -c
+          - |
+            . vpc-terraform-outputs/tfvars.sh
+            cd paas-cf/terraform/concourse
+            terraform remote config -backend=s3 -backend-config={{tfstate_bucket}} -backend-config="key=concourse.tfstate"
+            terraform apply -var env={{deploy_env}}
+
+    - task: concourse-terraform-outputs
+      config:
+        image: docker:///governmentpaas/bosh-init
+        inputs:
+        - name: paas-cf
+        - name: concourse-terraform-state
+        outputs:
+        - name: concourse-terraform-outputs
+        run:
+          path: sh
+          args:
+          - -e
+          - -c
+          - |
+            ruby paas-cf/concourse/scripts/extract_terraform_state_to_yaml.rb \
+            < concourse-terraform-state/concourse.tfstate \
+            > concourse-terraform-outputs/concourse-terraform-outputs.yml
+            ls -l concourse-terraform-outputs/concourse-terraform-outputs.yml
+            cat concourse-terraform-outputs/concourse-terraform-outputs.yml
+
+    - task: create-concourse-secrets
+      config:
+        params: {aws_access_key_id: {{aws_access_key_id}} }
+        params: {aws_secret_access_key: {{aws_secret_access_key}} }
+        params: {concourse_atc_password: {{concourse_atc_password}} }
+        params: {concourse_db_password: {{concourse_db_password}} }
+        params: {private_ssh_key: {{private_ssh_key}} }
+        outputs:
+          - name: concourse-secrets
+        run:
+          path: sh
+          args:
+          - -e
+          - -c
+          - |
+            cat > concourse-secrets/concourse-secrets.yml << EOF
+            ---
+            secrets:
+              aws_access_key_id: ${aws_access_key_id}
+              aws_secret_access_key: ${aws_secret_access_key}
+              concourse_atc_password: ${concourse_atc_password}
+              concourse_db_password: ${concourse_db_password}
+            EOF
+            ls -l concourse-secrets/concourse-secrets.yml
+            echo -n "${private_ssh_key}" > concourse-secrets/id_rsa
+            chmod 400 concourse-secrets/id_rsa
+            ls -l concourse-secrets/id_rsa
+
+    - task: spruce-up-concourse-manifest
```

Comment:
"spruce-up" in our terminology means to break st. down into pieces which can then be spruced back together. I'd simply call this generate-concourse-manifest, or merge-concourse-manifest